pkg/api/query/v1.go

```diff
@@ -52,6 +52,14 @@ import (
 	"github.com/thanos-io/thanos/pkg/tracing"
 )
 
+const (
+	dedupParam               = "dedup"
+	partialResponseParam     = "partial_response"
+	maxSourceResolutionParam = "max_source_resolution"
+	replicaLabelsParam       = "replicaLabels[]"
+	storeMatcherParam        = "storeMatch[]"
```

Comment:
Should we move this somewhere else or export it because we could use it in:
```
pkg/queryfrontend/codec.go:     result.StoreMatchers, err = parseStoreMatchersParam(r.Form["storeMatch[]"])
pkg/queryfrontend/codec.go:             params["storeMatch[]"] = storeMatchers
pkg/queryfrontend/codec.go:                     return nil, httpgrpc.Errorf(http.StatusBadRequest, errCannotParse, "storeMatch[]")
pkg/queryfrontend/codec.go:                     return nil, httpgrpc.Errorf(http.StatusBadRequest, "storeMatch[]")
```
?

Reply:
Agree. Good to export these.